### PR TITLE
Kevin branch

### PR DESCRIPTION
**Pull request by KevinLBrown806**

### CODE REVIEW

File: ADTA_5940_Team1_Correlational_Analysis.qmd
Comment (Tochiagbara) [inline ADTA_5940_Team1_Correlational_Analysis.qmd:1]: Where do I click in order to add analysis items.

**Review by KHarke — APPROVED** [review]

**Comment by KevinLBrown806** [comment]
latest

File: ADTA_5940_Team1_Correlational_Analysis.qmd
Comment (KevinLBrown806) [reply ADTA_5940_Team1_Correlational_Analysis.qmd:1]: resolved

**Review by KHarke — APPROVED** [review]
I will integrate this into Team_1.qmd.

**Pull request closed without merging**